--- archive/2023-01-07	2023-02-11 23:54:55
+++ archive/2022-11-07	2023-02-11 23:54:49
@@ -500,7 +500,7 @@
 499
 500 https://www.leatherman.com/skeletool-kb-500.html
 501 https://www.leatherman.com/skeletool-kbx-501.html
-502 https://www.leatherman.com/retired-products.html
+502 https://www.leatherman.com/juice-b2-502.html
 503 https://www.leatherman.com/juice-cs3-503.html
 504 https://www.leatherman.com/style-ps-25.html
 505
@@ -691,10 +691,25 @@
 690 https://www.leatherman.com/batch-003-parts-690.html
 691 https://www.leatherman.com/leatherman-tool-maintenance-kit-691.html
 692 https://www.leatherman.com/leatherman-blade-sharpener-692.html
-693
-694
+693 https://www.leatherman.com/batch-004-harvest-693.html
+694 https://www.leatherman.com/harvest-sheath-694.html
 695 https://www.leatherman.com/original-multi-tool-zip-up-hoodie-695.html
 696 https://www.leatherman.com/leatherman-logo-knit-cuffed-beanie-696.html
-697
+697 https://www.leatherman.com/europe-75-697.html
 698 https://www.leatherman.com/europe75-698.html
-699
+699 https://www.leatherman.com/40th-anniversary-699.html
+700
+701
+702
+703
+704
+705
+706
+707
+708
+709
+710
+711
+712
+713
+714
