--- archive/2023-02-10	2023-02-11 23:55:00
+++ archive/2023-01-07	2023-02-11 23:54:55
@@ -359,10 +359,10 @@
 358 https://www.leatherman.com/retired-products.html
 359 https://www.leatherman.com/30th-anniversary-micra-359.html
 360 https://www.leatherman.com/30th-anniv-super-tool-300-360.html
-361 https://www.leatherman.com/retired-products.html
+361 https://www.leatherman.com/juice-xe6-361.html
 362 https://www.leatherman.com/retired-products.html
 363 https://www.leatherman.com/retired-products.html
-364 https://www.leatherman.com/retired-products.html
+364 https://www.leatherman.com/juice-cs4-364.html
 365
 366
 367
@@ -691,20 +691,10 @@
 690 https://www.leatherman.com/batch-003-parts-690.html
 691 https://www.leatherman.com/leatherman-tool-maintenance-kit-691.html
 692 https://www.leatherman.com/leatherman-blade-sharpener-692.html
-693 https://www.leatherman.com/batch-006-harvest-693.html
-694 https://www.leatherman.com/harvest-sheath-694.html
+693
+694
 695 https://www.leatherman.com/original-multi-tool-zip-up-hoodie-695.html
 696 https://www.leatherman.com/leatherman-logo-knit-cuffed-beanie-696.html
-697 https://www.leatherman.com/batch-004-europe75-697.html
-698
-699 https://www.leatherman.com/40th-anniversary-tool-699.html
-700 https://www.leatherman.com/40th-anniversary-sheath-700.html
-701 https://www.leatherman.com/leatherman-adapt-kit-701.html
-702 https://www.leatherman.com/arc-702.html
-703
-704
-705
-706
-707
-708
-709
+697
+698 https://www.leatherman.com/europe75-698.html
+699
