--- archive/2022-07-20	2023-02-11 23:54:44
+++ archive/2023-02-03	2023-02-11 23:54:58
@@ -2,16 +2,16 @@
 1 https://www.leatherman.com/crunch-1.html
 2 https://www.leatherman.com/super-tool-300-2.html
 3 https://www.leatherman.com/surge-3.html
-4 /retired-products.html
-5 /retired-products.html
-6 /retired-products.html
+4 https://www.leatherman.com/retired-products.html
+5 https://www.leatherman.com/retired-products.html
+6 https://www.leatherman.com/retired-products.html
 7 https://www.leatherman.com/charge-tti-7.html
 8 https://www.leatherman.com/rebar-8.html
 9 https://www.leatherman.com/sidekick-9.html
 10 https://www.leatherman.com/wave-10.html
 11 https://www.leatherman.com/wingman-11.html
-12 /retired-products.html
-13 /retired-products.html
+12 https://www.leatherman.com/retired-products.html
+13 https://www.leatherman.com/retired-products.html
 14 https://www.leatherman.com/original-juice-c2-14.html
 15 https://www.leatherman.com/original-juice-cs4-15.html
 16 https://www.leatherman.com/original-juice-s2-16.html
@@ -19,18 +19,18 @@
 18 https://www.leatherman.com/skeletool-18.html
 19 https://www.leatherman.com/skeletool-cx-19.html
 20 https://www.leatherman.com/micra-20.html
-21 /retired-products.html
+21 https://www.leatherman.com/retired-products.html
 22 https://www.leatherman.com/squirt-ps4-22.html
-23 /retired-products.html
-24 /style-ps-25.html
+23 https://www.leatherman.com/retired-products.html
+24 https://www.leatherman.com/style-ps-25.html
 25 https://www.leatherman.com/style-ps-25.html
 26 https://www.leatherman.com/crater-c33-26.html
-27 /crater-c33-26.html
+27 https://www.leatherman.com/crater-c33-26.html
 28 https://www.leatherman.com/crater-c33t-28.html
 29 https://www.leatherman.com/mut-29.html
 30 https://www.leatherman.com/mut-eod-30.html
 31 https://www.leatherman.com/super-tool-300-eod-31.html
-32 /retired-products.html
+32 https://www.leatherman.com/retired-products.html
 33
 34
 35
@@ -44,23 +44,23 @@
 43
 44
 45
-46 /pockettools
-47 /pockettools
-48 /retired-products.html
+46 https://www.leatherman.com/pockettools
+47 https://www.leatherman.com/pockettools
+48 https://www.leatherman.com/retired-products.html
 49 https://www.leatherman.com/oht-49.html
 50
 51 https://www.leatherman.com/raptor-rescue-51.html
 52 https://www.leatherman.com/crater-c33lx-52.html
 53 https://www.leatherman.com/crater-c33x-53.html
 54 https://www.leatherman.com/crater-c33tx-54.html
-55 /pocket-size-multi-tools
+55 https://www.leatherman.com/pocket-size-multi-tools
 56 https://www.leatherman.com/hail-56.html
 57
 58
 59
 60
 61
-62 /retired-products.html
+62 https://www.leatherman.com/retired-products.html
 63
 64
 65
@@ -68,38 +68,38 @@
 67
 68
 69
-70 /pockettools
-71 /pockettools
-72 /pockettools
+70 https://www.leatherman.com/pockettools
+71 https://www.leatherman.com/pockettools
+72 https://www.leatherman.com/pockettools
 73 https://www.leatherman.com/raptor-rescue-holster-73.html
 74
 75
 76
-77 /retired-products.html
-78 /retired-products.html
+77 https://www.leatherman.com/retired-products.html
+78 https://www.leatherman.com/retired-products.html
 79 https://www.leatherman.com/original-wave-79.html
-80 /retired-products.html
-81 /retired-products.html
-82 /retired-products.html
-83 /retired-products.html
-84 /retired-products.html
-85 /super-tool-300-2.html
-86 /super-tool-300-2.html
-87 /retired-products.html
-88 /retired-products.html
-89 /retired-products.html
-90 /retired-products.html
-91 /retired-products.html
-92 /retired-products.html
-93 /retired-products.html
+80 https://www.leatherman.com/retired-products.html
+81 https://www.leatherman.com/retired-products.html
+82 https://www.leatherman.com/retired-products.html
+83 https://www.leatherman.com/retired-products.html
+84 https://www.leatherman.com/retired-products.html
+85 https://www.leatherman.com/super-tool-300-2.html
+86 https://www.leatherman.com/super-tool-300-2.html
+87 https://www.leatherman.com/retired-products.html
+88 https://www.leatherman.com/retired-products.html
+89 https://www.leatherman.com/retired-products.html
+90 https://www.leatherman.com/retired-products.html
+91 https://www.leatherman.com/retired-products.html
+92 https://www.leatherman.com/retired-products.html
+93 https://www.leatherman.com/retired-products.html
 94
-95 /knifeless-fuse-97.html
+95 https://www.leatherman.com/knifeless-fuse-97.html
 96 https://www.leatherman.com/kick-96.html
-97 /retired-products.html
+97 https://www.leatherman.com/retired-products.html
 98
-99 /retired-products.html
-100 /retired-products.html
-101 /retired-products.html
+99 https://www.leatherman.com/retired-products.html
+100 https://www.leatherman.com/retired-products.html
+101 https://www.leatherman.com/retired-products.html
 102
 103
 104
@@ -118,10 +118,10 @@
 117
 118
 119
-120 /retired-products.html
-121 /retired-products.html
-122 /retired-products.html
-123 /retired-products.html
+120 https://www.leatherman.com/retired-products.html
+121 https://www.leatherman.com/retired-products.html
+122 https://www.leatherman.com/retired-products.html
+123 https://www.leatherman.com/retired-products.html
 124
 125
 126 https://www.leatherman.com/removable-bit-driver-126.html
@@ -198,66 +198,66 @@
 197
 198 https://www.leatherman.com/black-molle-sheath-198.html
 199 https://www.leatherman.com/black-logo-t-shirt-199.html
-200 /retired-products.html
+200 https://www.leatherman.com/retired-products.html
 201
 202 https://www.leatherman.com/leather-box-sheath-4-2-202.html
-203 /retired-products.html
-204 /retired-products.html
-205 /retired-products.html
-206 /retired-products.html
-207 /retired-products.html
+203 https://www.leatherman.com/retired-products.html
+204 https://www.leatherman.com/retired-products.html
+205 https://www.leatherman.com/retired-products.html
+206 https://www.leatherman.com/retired-products.html
+207 https://www.leatherman.com/retired-products.html
 208 https://www.leatherman.com/baseball-logo-hat-208.html
 209 https://www.leatherman.com/black-logo-beanie-209.html
-210 /retired-products.html
-211 /retired-products.html
-212 /retired-products.html
-213 /retired-products.html
-214 /retired-products.html
+210 https://www.leatherman.com/retired-products.html
+211 https://www.leatherman.com/retired-products.html
+212 https://www.leatherman.com/retired-products.html
+213 https://www.leatherman.com/retired-products.html
+214 https://www.leatherman.com/retired-products.html
 215 https://www.leatherman.com/gray-flexfit-baseball-logo-hat-215.html
 216 https://www.leatherman.com/gray-oregon-proud-t-shirt-216.html
 217
-218 /retired-products.html
-219 /retired-products.html
-220 /retired-products.html
-221 /retired-products.html
-222 /retired-products.html
-223 /retired-products.html
-224 /retired-products.html
-225 /retired-products.html
-226 /retired-products.html
-227 /retired-products.html
-228 /retired-products.html
-229 /retired-products.html
-230 /retired-products.html
+218 https://www.leatherman.com/retired-products.html
+219 https://www.leatherman.com/retired-products.html
+220 https://www.leatherman.com/retired-products.html
+221 https://www.leatherman.com/retired-products.html
+222 https://www.leatherman.com/retired-products.html
+223 https://www.leatherman.com/retired-products.html
+224 https://www.leatherman.com/retired-products.html
+225 https://www.leatherman.com/retired-products.html
+226 https://www.leatherman.com/retired-products.html
+227 https://www.leatherman.com/retired-products.html
+228 https://www.leatherman.com/retired-products.html
+229 https://www.leatherman.com/retired-products.html
+230 https://www.leatherman.com/retired-products.html
 231
-232 /retired-products.html
-233 /retired-products.html
-234 /retired-products.html
-235 /retired-products.html
-236 /retired-products.html
-237 /sheaths
+232 https://www.leatherman.com/retired-products.html
+233 https://www.leatherman.com/retired-products.html
+234 https://www.leatherman.com/retired-products.html
+235 https://www.leatherman.com/retired-products.html
+236 https://www.leatherman.com/retired-products.html
+237 https://www.leatherman.com/sheaths
 238 https://www.leatherman.com/mut-skull-t-shirt-238.html
-239 /retired-products.html
-240 /retired-products.html
-241 /retired-products.html
-242 /retired-products.html
-243 /style-ps-25.html
+239 https://www.leatherman.com/retired-products.html
+240 https://www.leatherman.com/retired-products.html
+241 https://www.leatherman.com/retired-products.html
+242 https://www.leatherman.com/retired-products.html
+243 https://www.leatherman.com/style-ps-25.html
 244
 245
 246 https://www.leatherman.com/clear-water-bottle-with-yellow-print-and-black-cap-246.html
 247
-248 /sheaths
+248 https://www.leatherman.com/sheaths
 249 https://www.leatherman.com/replacement-bits-for-mako-ti-249.html
 250 https://www.leatherman.com/mut-m4-t-shirt-250.html
-251 /retired-products.html
-252 /retired-products.html
-253 /retired-products.html
-254 /retired-products.html
-255 /retired-products.html
+251 https://www.leatherman.com/retired-products.html
+252 https://www.leatherman.com/retired-products.html
+253 https://www.leatherman.com/retired-products.html
+254 https://www.leatherman.com/retired-products.html
+255 https://www.leatherman.com/retired-products.html
 256 https://www.leatherman.com/bits-for-original-piranha-256.html
 257 https://www.leatherman.com/leather-box-sheath-4-257.html
 258 https://www.leatherman.com/logo-mouse-pad-258.html
-259 /sheaths
+259 https://www.leatherman.com/sheaths
 260
 261 https://www.leatherman.com/wingman-t-shirt-261.html
 262 https://www.leatherman.com/strap-cutter-for-z-rex-262.html
@@ -305,18 +305,18 @@
 304
 305
 306 https://www.leatherman.com/carbon-scraper-306.html
-307 /sheaths
+307 https://www.leatherman.com/sheaths
 308 https://www.leatherman.com/leather-box-sheath-4-5-308.html
-309 /sheaths
+309 https://www.leatherman.com/sheaths
 310 https://www.leatherman.com/leather-sleeve-for-sidekick-and-wingman-310.html
 311 https://www.leatherman.com/original-wave-heritage-sheath-311.html
 312
 313 https://www.leatherman.com/premium-leather-sheath-4-313.html
-314 /sheaths
-315 /sheaths
-316 /black-molle-sheath-198.html
-317 /brown-molle-sheath-405.html
-318 /z-rex-32.html
+314 https://www.leatherman.com/sheaths
+315 https://www.leatherman.com/sheaths
+316 https://www.leatherman.com/black-molle-sheath-198.html
+317 https://www.leatherman.com/brown-molle-sheath-405.html
+318 https://www.leatherman.com/z-rex-32.html
 319
 320
 321
@@ -330,19 +330,19 @@
 329 https://www.leatherman.com/wave-shaped-dog-toy-329.html
 330 https://www.leatherman.com/womens-black-logo-t-shirt-330.html
 331
-332 /surge-3.html
+332 https://www.leatherman.com/surge-3.html
 333 https://www.leatherman.com/heritage-pst-leather-sheath-333.html
-334 /retired-products.html
-335 /retired-products.html
-336 /retired-products.html
-337 /pockettools
+334 https://www.leatherman.com/retired-products.html
+335 https://www.leatherman.com/retired-products.html
+336 https://www.leatherman.com/retired-products.html
+337 https://www.leatherman.com/pockettools
 338 https://www.leatherman.com/shooter-338.html
 339 https://www.leatherman.com/original-juice-sx-339.html
-340 /crater-c33x-53.html
+340 https://www.leatherman.com/crater-c33x-53.html
 341
 342 https://www.leatherman.com/premium-knife-nylon-sheath-i-342.html
 343 https://www.leatherman.com/premium-knife-nylon-sheath-ii-343.html
-344 /pockettools
+344 https://www.leatherman.com/pockettools
 345
 346
 347
@@ -356,13 +356,13 @@
 355
 356
 357
-358 /retired-products.html
+358 https://www.leatherman.com/retired-products.html
 359 https://www.leatherman.com/30th-anniversary-micra-359.html
 360 https://www.leatherman.com/30th-anniv-super-tool-300-360.html
-361 https://www.leatherman.com/juice-xe6-361.html
-362 /retired-products.html
-363 /retired-products.html
-364 https://www.leatherman.com/juice-cs4-364.html
+361 https://www.leatherman.com/retired-products.html
+362 https://www.leatherman.com/retired-products.html
+363 https://www.leatherman.com/retired-products.html
+364 https://www.leatherman.com/retired-products.html
 365
 366
 367
@@ -396,7 +396,7 @@
 395 https://www.leatherman.com/leatherman-1-395.html
 396 https://www.leatherman.com/leatherman-2-396.html
 397 https://www.leatherman.com/leatherman-3-397.html
-398 /retired-products.html
+398 https://www.leatherman.com/retired-products.html
 399 https://www.leatherman.com/leatherman-5-399.html
 400 https://www.leatherman.com/leatherman-6-400.html
 401 https://www.leatherman.com/leatherman-7-401.html
@@ -447,7 +447,7 @@
 446 https://www.leatherman.com/link-1-446.html
 447 https://www.leatherman.com/link-2-447.html
 448 https://www.leatherman.com/link-3-448.html
-449 /
+449 https://www.leatherman.com/
 450 https://www.leatherman.com/link-6-450.html
 451 https://www.leatherman.com/link-7-451.html
 452 https://www.leatherman.com/link-8-452.html
@@ -486,7 +486,7 @@
 485 https://www.leatherman.com/link-17-485.html
 486 https://www.leatherman.com/link-18-486.html
 487 https://www.leatherman.com/link-19-487.html
-488 /tread-425.html
+488 https://www.leatherman.com/tread-425.html
 489 https://www.leatherman.com/xeo-19r-w-accessory-kit-489.html
 490
 491
@@ -500,9 +500,9 @@
 499
 500 https://www.leatherman.com/skeletool-kb-500.html
 501 https://www.leatherman.com/skeletool-kbx-501.html
-502 https://www.leatherman.com/juice-b2-502.html
+502 https://www.leatherman.com/retired-products.html
 503 https://www.leatherman.com/juice-cs3-503.html
-504 /style-ps-25.html
+504 https://www.leatherman.com/style-ps-25.html
 505
 506
 507
@@ -517,7 +517,7 @@
 516 https://www.leatherman.com/tecopatx-516.html
 517 https://www.leatherman.com/tecopa-tx-517.html
 518 https://www.leatherman.com/charge-al-prep-kit-518.html
-519 /retired-products.html
+519 https://www.leatherman.com/retired-products.html
 520 https://www.leatherman.com/link-20-520.html
 521
 522
@@ -530,13 +530,13 @@
 529
 530
 531 https://www.leatherman.com/tread-lt-531.html
-532 /heritage-pst-553.html
+532 https://www.leatherman.com/heritage-pst-553.html
 533
-534 /wave-10.html
+534 https://www.leatherman.com/wave-10.html
 535 https://www.leatherman.com/old-charge-alx-535.html
 536 https://www.leatherman.com/standard-nylon-sheath-536.html
-537 /charge-tti-7.html?cgid=multi-tools
-538 /wave-10.html
+537 https://www.leatherman.com/charge-tti-7.html?cgid=multi-tools
+538 https://www.leatherman.com/wave-10.html
 539
 540
 541
@@ -544,7 +544,7 @@
 543
 544
 545
-546 https://www.leatherman.com/black-silver-wave-546.html
+546 https://www.leatherman.com/wave-10.html
 547
 548 https://www.leatherman.com/nylon-sheath-548.html
 549 https://www.leatherman.com/nylon-sheath-w-pockets-549.html
@@ -553,29 +553,29 @@
 552 https://www.leatherman.com/charge-552.html
 553 https://www.leatherman.com/heritage-pst-553.html
 554
-555 /on/demandware.store/Sites-lmusa-Site/default/Product-Show?pid=multi-tools
+555 https://www.leatherman.com/on/demandware.store/Sites-lmusa-Site/default/Product-Show?pid=multi-tools
 556 https://www.leatherman.com/heritage-rebar-556.html
 557 https://www.leatherman.com/heritage-wave-557.html
 558 https://www.leatherman.com/heritage-super-tool-300-558.html
 559 https://www.leatherman.com/heritage-micra-559.html
 560 https://www.leatherman.com/heritage-charge-560.html
-561 /crunch-1.html
+561 https://www.leatherman.com/crunch-1.html
 562 https://www.leatherman.com/tread-tempo-kit-562.html
 563
 564
 565
 566 https://www.leatherman.com/heritage-sheath-566.html
-567 /surge-3.html
-568 https://www.leatherman.com/black-silver-signal-568.html
-569 https://www.leatherman.com/black-silver-skeletool-569.html
-570 https://www.leatherman.com/black-silver-skeletool-kbx-570.html
-571 /retired-products.html
-572 /skeletool
+567 https://www.leatherman.com/surge-3.html
+568 https://www.leatherman.com/signal-439.html
+569 https://www.leatherman.com/skeletool-18.html
+570 https://www.leatherman.com/skeletool-kbx-501.html
+571 https://www.leatherman.com/retired-products.html
+572 https://www.leatherman.com/skeletool
 573 https://www.leatherman.com/lanyard-ring-pocket-clip-573.html
 574 https://www.leatherman.com/tread-tempo-lt-574.html
-575 /rebar-8.html
-576 /signal-439.html
-577 /skeletool-18.html
+575 https://www.leatherman.com/rebar-8.html
+576 https://www.leatherman.com/signal-439.html
+577 https://www.leatherman.com/skeletool-18.html
 578 https://www.leatherman.com/coyote-skeletool-kbx-578.html
 579 https://www.leatherman.com/free-t2-579.html
 580 https://www.leatherman.com/free-t3-580.html
@@ -591,10 +591,10 @@
 590 https://www.leatherman.com/free-k4-590.html
 591 https://www.leatherman.com/free-k4x-591.html
 592 https://www.leatherman.com/free-k5-not-used-592.html
-593 https://www.leatherman.com/skeletool-topo-593.html
-594 /skeletool-18.html
+593 https://www.leatherman.com/skeletool-18.html
+594 https://www.leatherman.com/skeletool-18.html
 595 https://www.leatherman.com/charge-forest-camo-595.html
-596 /charge-552.html
+596 https://www.leatherman.com/charge-552.html
 597
 598
 599
@@ -615,7 +615,7 @@
 614
 615
 616 https://www.leatherman.com/ainsworth-premium-leather-sheath-616.html
-617 /retired-products.html
+617 https://www.leatherman.com/retired-products.html
 618 https://www.leatherman.com/realtree-camo-sheath-4-5-618.html
 619 https://www.leatherman.com/gift-box-619.html
 620 https://www.leatherman.com/carbide-glass-breaker-bit-620.html
@@ -635,29 +635,29 @@
 634 https://www.leatherman.com/charge-damascus-carbon-634.html
 635 https://www.leatherman.com/charge-damascus-635.html
 636 https://www.leatherman.com/charge-g10-636.html
-637 /style-ps-25.html
+637 https://www.leatherman.com/style-ps-25.html
 638 https://www.leatherman.com/2h-wave-638.html
 639 https://www.leatherman.com/2-handed-charge-alx-639.html
 640 https://www.leatherman.com/2-handed-charge-al-640.html
 641 https://www.leatherman.com/2-handed-charge-tti-641.html
 642 https://www.leatherman.com/micra-642.html
 643 https://www.leatherman.com/sight-adjuster-bit-driver-extender-643.html
-644 /free-t2-579.html
+644 https://www.leatherman.com/free-t2-579.html
 645 https://www.leatherman.com/bond-645.html
 646 https://www.leatherman.com/curl-646.html
 647 https://www.leatherman.com/bolster-647.html
 648 https://www.leatherman.com/style-ps-gift-box-648.html
-649 /crater-c33x-53.html
-650 /
-651 /
-652 /
+649 https://www.leatherman.com/crater-c33x-53.html
+650 https://www.leatherman.com/
+651 https://www.leatherman.com/
+652 https://www.leatherman.com/
 653 https://www.leatherman.com/clean-contact-carabiner-653.html
-654 /multi-tools
+654 https://www.leatherman.com/multi-tools
 655 https://www.leatherman.com/limited-edition-raptor-655.html
-656 /outdoor
+656 https://www.leatherman.com/outdoor
 657 https://www.leatherman.com/wave-repair-kit-657.html
-658 /preparedness
-659 /clean-contact-carabiner-653.html
+658 https://www.leatherman.com/preparedness
+659 https://www.leatherman.com/clean-contact-carabiner-653.html
 660 https://www.leatherman.com/new-era-trucker-cap-660.html
 661 https://www.leatherman.com/raptor-response-661.html
 662 https://www.leatherman.com/pst-heritage-short-sleeve-tee-662.html
@@ -691,10 +691,13 @@
 690 https://www.leatherman.com/batch-003-parts-690.html
 691 https://www.leatherman.com/leatherman-tool-maintenance-kit-691.html
 692 https://www.leatherman.com/leatherman-blade-sharpener-692.html
-693 https://www.leatherman.com/harvest-693.html
+693
 694 https://www.leatherman.com/harvest-sheath-694.html
 695 https://www.leatherman.com/original-multi-tool-zip-up-hoodie-695.html
 696 https://www.leatherman.com/leatherman-logo-knit-cuffed-beanie-696.html
 697
 698
-699
+699 https://www.leatherman.com/40th-anniversary-tool-699.html
+700 https://www.leatherman.com/40th-anniversary-sheath-700.html
+701 https://www.leatherman.com/leatherman-adapt-kit-701.html
+702 https://www.leatherman.com/arc-702.html
